x-pack/plugins/observability/common/typings.ts

```diff
@@ -25,3 +25,13 @@ export interface ApmIndicesConfig {
   apmAgentConfigurationIndex: string;
   apmCustomLinkIndex: string;
 }
+export enum AlertStatus {
+  All = 'all',
+  Active = 'active',
```

Comment:
@fkanout We have already defined the `active` and `recovered` values here `kibana/packages/kbn-rule-data-utils/src/alerts_as_data_status.ts`, so I suggest you don't hardcode the values and you import from the existing package.

You can import the values like this `import { ALERT_STATUS_ACTIVE, ALERT_STATUS_RECOVERED } from '@kbn/rule-data-utils/alerts_as_data_status';`

Reply:
That's cool. I didn't know that. Updated accordingly